src/store/savedSet/index.ts

```diff
@@ -1,13 +1,18 @@
 import { useSelector } from 'react-redux';
-import { savedSetSelector } from './selector';
-import EnvironmentVariables from '../../helpers/EnvVariables';
+import { singularizeFilesSetType } from 'views/DataExploration/components/SetsManagementDropdown';
+
 import { SetType } from 'services/api/savedSet/models';
 
+import EnvironmentVariables from '../../helpers/EnvVariables';
+
+import { savedSetSelector } from './selector';
+
 export type { initialState as SavedSetInitialState } from './types';
 export { default, SavedSetState } from './slice';
 
 export const MAX_LENGTH_NAME = 50;
-export const getSetFieldId = (type: SetType) => `${type}_facet_ids.${type}_fhir_id_1`;
+export const getSetFieldId = (type: SetType) =>
+  `${singularizeFilesSetType(type)}_facet_ids.${singularizeFilesSetType(type)}_fhir_id_1`;
```

Comment:
I don't really like this solution but I don't see any other way, also was wondering if ```singularizeFilesSetType```should be moved to a utils file somewhere else

Reply:
You are right, but I think it is fine. Perhaps we could remove the dependence of `type` on the way we name our indices.